assets/blocks/lesson-actions/lesson-actions-block/settings.js

```diff
@@ -0,0 +1,24 @@
+import { InspectorControls } from '@wordpress/block-editor';
+import { PanelBody, ToggleControl } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Inspector controls for lesson actions block.
+ *
+ * @param {Object}   props
+ * @param {Object}   props.activeBlocks Active blocks, where the key is the block name.
+ * @param {Function} props.toggleBlock  Toggle block.
+ */
+export const LessonActionsBlockSettings = ( { activeBlocks, toggleBlock } ) => (
+	<InspectorControls>
+		<PanelBody title={ __( 'Additional Actions', 'sensei-lms' ) }>
+			<ToggleControl
+				checked={
+					false !== activeBlocks[ 'sensei-lms/button-reset-lesson' ]
+				}
+				onChange={ toggleBlock( 'sensei-lms/button-reset-lesson' ) }
+				label={ __( 'Reset lesson', 'sensei-lms' ) }
```

Comment:
I find the label a bit confusing. We could rename this to "Enable 'Reset lesson'" or "'Reset lesson' enabled"

Reply:
I don't know if we need that, considering we have the "Additional Actions" title. WDYT @donnapep and @yscik?

Reply:
Taking a look at other core blocks like _Gallery_ or _Cover_, they don't prepend "Enable" to the labels. Implicitly most people understand that a toggle enables or disables something.